core/bones/file.py

```diff
@@ -189,3 +189,8 @@ def recreateFileEntryIfNeeded(val):
                     return
                 importBlobFromViur2(val["dest"]["dlkey"], val["dest"]["name"])
                 recreateFileEntryIfNeeded(val["dest"])
+
+    def structure(self) -> dict:
+        return super().structure() | {
+            "validMimeTypes": self.validMimeTypes
```

Comment:
This is still camelCase?

Maybe replace by
```suggestion
            "valid_mime_types": self.validMimeTypes
```
or 

```suggestion
            "validmimetypes": self.validMimeTypes
```
?